libraries/core_libs/consensus/src/dag/sortition_params_manager.cpp

```diff
@@ -126,13 +153,22 @@ uint16_t SortitionParamsManager::averageCorrectionPerPercent() const {
 }
 
 void SortitionParamsManager::cleanup(uint64_t current_period) {
-  dag_efficiencies_.clear();
+  uint16_t efficiencies_to_leave = 0;
+  if (current_period >= k_testnet_hardfork2_block_num) {
+    efficiencies_to_leave = k_testnet_hardfork2_sortition_interval_length - config_.computation_interval;
```

Comment:
as mentioned above, if I understand it correctly, config_.computation_interval is now changing_interval and is computation_interval - confusing wording here